source/1_engine/rex_engine/src/memory/global_allocator.cpp

```diff
@@ -1,12 +1,41 @@
 #include "rex_engine/memory/global_allocator.h"
 
+#include "rex_engine/memory/memory_stats.h"
+#include "rex_engine/diagnostics/debug.h"
+
 namespace rex
 {
+  namespace internal
+  {
+    bool log_mem_usage_before_tracking_impl()
+    {
+      log_mem_usage();
+      return true;
+    }
+
+    void log_mem_usage_before_tracking()
+    {
+      // Store result in static bool so we only log once
+      static bool _ = log_mem_usage_before_tracking_impl();
```

Comment:
Can you give this local a proper name please.
I don't like this introduction of C# using _ as a temp variable name, please refrain from using it in C++.